examples/generative/pixelcnn.py

```diff
@@ -0,0 +1,168 @@
+"""
+Title: PixelCNN
+Author: [ADMoreau](https://github.com/ADMoreau)
+Date created: 2020/05/17
+Last modified: 2020/05/23
+Description: PixelCNN implemented in Keras.
+"""
```

Comment:
Please add an `## Introduction` section with:
1. a paragraph explaining what PixelCNN is and what it's useful for
2. a link to the reference paper